### PR TITLE
testing tmp1

### PR DESCRIPTION
**Pull request by Kamill90**

### CODE REVIEW

File: src/screens/HomeScreen.test.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    };
+
+    const screen = render(<HomeScreen {...props} />);
+    await waitFor(() => {
+      const wroclawItem = screen.getByTestId("list-item-Wroclaw");
+      expect(wroclawItem).toBeTruthy();
+      fireEvent.press(wroclawItem);
+
+      const expectedCityData = normalizeWeatherData(groupData.list[1]);
+      console.log({ expectedCityData });
```
Comment (adamTrz) [inline src/screens/HomeScreen.test.tsx:42]: 😉

File: src/screens/HomeScreen.test.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      expect(props.navigation.navigate).toHaveBeenCalledWith("Details", {
+        item: expectedCityData,
+      });
+    });
+  });
+
+  it("display only ActivityIndicator in case of 400 status in response", async () => {
```
Comment (adamTrz) [inline src/screens/HomeScreen.test.tsx:49]: Shouldn't we show some error message instead and not infinite spinner? 🤔

File: src/network/mock.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  ],
+};
+
+export const getCurrentWeatherForGroupMock200 = () =>
+  mock.onGet("group").reply(200, groupData);
+
+export const getCurrentWeatherForGroupMock400 = () =>
+  mock.onGet("group").reply(400);
+
+// export const postLoginMock412 = () => mock.onPost("/auth/login").reply(412);
```
Comment (adamTrz) [inline src/network/mock.ts:70]: Please remove dead code :)

**Review by adamTrz — APPROVED** [review]
Left some nits and comments :)

File: src/screens/HomeScreen.test.tsx (hunk 2 above)
Comment (Kamill90) [reply src/screens/HomeScreen.test.tsx:49]: There is an alert. It isn't tested tho.

**Pull request merged**